themes/socialbase/templates/swiftmailer.html.twig

```diff
@@ -359,7 +359,13 @@
                             <table border="0" cellpadding="0" cellspacing="0">
                                 <tr>
                                     <td>
+                                        {% if header %}
+                                            {{ header }}
+                                        {% endif %}
                                         {{ body }}
+                                        {% if footer %}
```

Comment:
If we set default empty values for our variables then we may not need the if statement here but can just always print it.